### DISCUSSION TITLE
Add Insyde fd packer?

### DISCUSSION THREAD

**Discussion by ShadeTechnik** [platomav/BIOSUtilities#48] [General]
Not sure if this is beyond the scope. I've been having trouble finding info about creating an fd file, there's plenty of info about extracting them. I've never directly used the Insyde tools so maybe it's a proprietary thing but devices I've been working on use h20fft to flash the bios with the fd files. So if I extract them, modify the binary, the only thing I know to do is write it directly to the flash chip with a hardware programmer, which can be a pain to do.

**Comment by platomav** [comment]
I don't understand the question or intent here. Add what? The Insyde FD Packer (or whatever it is called) is a proprietary/paid/NDA utility for vendors to generate update packages. There is a 3rd party utility in this repo to extract those Insyde packages into their components. That's all.

**Reply by ShadeTechnik** [reply]
Well I think you understood the question quite well. I wasn't sure the utility to create fd files was proprietary on that level, which is what I asked, and you answered. Whether an "Open Source" utility could be written to create them, isn't understood but I understand from your reply that there is no intent on doing so.

**Reply by platomav** [reply]
Ah, ok. I guess one could write such a utility to build instead of extract. Indeed, there is no intent to write such a utility as part of this repository. 👍
